test/Syntax/tokens_escaped_identifier.swift

```diff
@@ -0,0 +1,4 @@
+// RUN: %swift-syntax-test -input-source-filename %s -dump-full-tokens > %t
```

Comment:
Essentially, `-dump-full-tokens` is for debug use only.
So could you change this to `FileCheck` based test?
like:
```
// CHECK-LABEL 3:25
// CHECK-NEXT: (token identifier
// CHECK-NEXT:  (trivia block_comment
// CHECK-NEXT:  (trivia space 1)
// CHECK-NEXT:  (trivia backtick 1)
// CHECK-NEXT:  (text="if")
// CHECK-NEXT:  (trivia backtick 1)

// CHECK-LABEL 4:27
// CHECK-NEXT: (token identifier
// CHECK-NEXT: ...
```

Reply:
OK, I will try it.